src/views/FindGroups.vue

```diff
@@ -41,9 +41,11 @@
                 <ion-icon :icon="ellipsisVerticalOutline" slot="icon-only"/>
               </ion-button>
             </ion-item>
-            <ion-item @click="viewFacilities(group.facilityGroupId)" button>
+            <ion-item button>
```

Comment:
We can remove button property as now we don't want the UI to show that the item is clickable.